src/formatting_stack/formatters/newlines.clj

```diff
@@ -1,20 +1,36 @@
 (ns formatting-stack.formatters.newlines
   (:require
+   [clojure.string :as str]
    [com.gfredericks.all-my-files-should-end-with-exactly-one-newline-character :as impl]
    [formatting-stack.protocols.formatter :as formatter]
-   [formatting-stack.util :refer [process-in-parallel!]]
+   [formatting-stack.protocols.linter :as linter]
+   [formatting-stack.util :refer [process-in-parallel! silence]]
    [nedap.speced.def :as speced]
    [nedap.utils.modular.api :refer [implement]]))
 
 (defn format! [{:keys [expected-newline-count]} files]
-  (with-out-str ;; Supress "All newlines are good, nothing to fix."
+  (silence ;; Supress "All newlines are good, nothing to fix."
```

Comment:
Maybe for both instances of `silence` in this pr, they'd be better placed around `impl/*`, and no broader.

That way it's clearer which part is at fault. Also I'd be slightly afraid of parallelism mixed with silencing (it has worked well for us, but seems delicate)